integration_tests/snaps/simple-svn/snapcraft.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-package
+version: 0.1
+summary: svn integration test
+description: use svn
+confinement: strict
+
+parts:
+    svn:
+        plugin: make
+        source: .
+        source-type: svn
```

Comment:
It would be nice to test "svn:" source lines, too, and one check-out and one update.